### ISSUE TITLE
Rule: vue/comment-directive

### ISSUE THREAD

**Issue by senpp** [nuxt/eslint-plugin-nuxt#122]
Hi everyone, I've caught an error about **clear** of **vue/comment-directive** after upgrading from version 1.0.0 to version 2.0.0.

```
10:12  error  clear  vue/comment-directive
```

### Versions
- nuxt v2.14.7
- node v14.15.0
- eslint-plugin-nuxt v2.0.0

**Comment by clarkdo** [comment]
Hi @senpp , look this rule has new change in eslint-plugin-vue v7. 

You can check [eslint-plugin-vue doc](https://github.com/vuejs/eslint-plugin-vue/blob/d4cd35ec550629a44dd66b3e397dad9218bea1a1/docs/rules/comment-directive.md) for more details.

**Comment by senpp** [comment]
> Hi @senpp , look this rule has new change in eslint-plugin-vue v7.
> 
> You can check [eslint-plugin-vue doc](https://github.com/vuejs/eslint-plugin-vue/blob/d4cd35ec550629a44dd66b3e397dad9218bea1a1/docs/rules/comment-directive.md) for more details.

Hi @clarkdo , I don't use any code comment in my file, but I've resolved this with [#121 ]. But I need to manually upgrade in my package.json

**Comment by RyanMulready** [comment]
I'm having the same issue. This error is being reported on files with no comments at all since updating to version 2. Downgrading back to 1 fixes the problem.

**Comment by clarkdo** [comment]
@RyanMulready I think you can try to upgrade `@nuxtjs/eslint-config` as @senpp mentioned for making sure `eslint` and `eslint-plugin-vue` are all updated .

**Comment by hendydwip** [comment]
Hi, wanna ask if anyone gets the same as I do like this after update the package `@nuxt/eslint_config` and `@nuxt/eslint_module` ?
![image](https://user-images.githubusercontent.com/45591084/100055369-500c9880-2e56-11eb-863f-aaf3309325ca.png)

**Comment by senpp** [comment]
> Hi, wanna ask if anyone gets the same as I do like this after update the package `@nuxt/eslint_config` and `@nuxt/eslint_module` ?
> ![image](https://user-images.githubusercontent.com/45591084/100055369-500c9880-2e56-11eb-863f-aaf3309325ca.png)

Have you tried something like that?
```
rm -rf node_modules && rm yarn.lock && yarn cache clean
```

**Comment by hendydwip** [comment]
> > Hi, wanna ask if anyone gets the same as I do like this after update the package `@nuxt/eslint_config` and `@nuxt/eslint_module` ?
> > ![image](https://user-images.githubusercontent.com/45591084/100055369-500c9880-2e56-11eb-863f-aaf3309325ca.png)
> 
> Have you tried something like that?
> 
> ```
> rm -rf node_modules && rm yarn.lock && yarn cache clean
> ```

Sorry for the late reply.
Yeah, I tried that, I'm using NPM right now. Still get the error somehow, but I tried to install `npm install eslint-loader`, and work perfectly hehehe

**Comment by RyanMulready** [comment]
> @RyanMulready I think you can try to upgrade `@nuxtjs/eslint-config` as @senpp mentioned for making sure `eslint` and `eslint-plugin-vue` are all updated .

Tried again today running the following command, still getting the same error

```
npm install @nuxtjs/eslint-config@latest eslint@latest eslint-plugin-vue@latest eslint-plugin-nuxt@latest
```

**Comment by RyanMulready** [comment]
I was eventually able to get this resolved though I'm not sure what I did differently... here is my package.json settings for relevant deps

```javascript
        "@babel/core": "^7.12.9",
        "@babel/eslint-parser": "^7.12.1",
        "@nuxtjs/eslint-module": "^3.0.1",
        "eslint-plugin-nuxt": "^2.0.0",
        "eslint-plugin-vue": "^7.1.0",
        "eslint": "^7.14.0",
```

**Comment by CristianLluis** [comment]
> I was eventually able to get this resolved though I'm not sure what I did differently... here is my package.json settings for relevant deps
> 
> ```js
>         "@babel/core": "^7.12.9",
>         "@babel/eslint-parser": "^7.12.1",
>         "@nuxtjs/eslint-module": "^3.0.1",
>         "eslint-plugin-nuxt": "^2.0.0",
>         "eslint-plugin-vue": "^7.1.0",
>         "eslint": "^7.14.0",
> ```

Updating eslint solved the issue for me. Thx 👍

**Comment by dimm-tech** [comment]
I have the same problem, I tried npm update, but it didn't help. there are no records in package.json about eslint. i'm use nuxt ^2.14.12

**Comment by vinstah** [comment]
I had an old project the when i installed node modules I got the same error.
I used yarn add nuxt to get the latest then yarn add @nuxtjs/eslint-module @nuxtjs/eslint-config

then no more error :)
